_data/api-swagger.yaml

```diff
@@ -546,6 +547,27 @@ definitions:
         }
       }
       ```
+
+  SigningCertificate:
+    allOf:
+      - $ref: '#/definitions/Verification'
+    example: |
+      To create a transaction to be signed with a qualified certificate (QC)
```

Comment:
The reason this is called Signing Certificate is because we don't require the use of a qualified certificate. A qualified certificate just happens to be one of the certificates which can be used.